analyzers/tests/SonarAnalyzer.UnitTest/ResourceTests/SonarLintXml/PropertiesCSharpTrueVbnetFalse/SonarLint.xml

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wasn't there a PR to remove usage of this directory?

Reply:
Not the whole directory.

The cleanup process did:
- remove all the files that were easily generated by the new methods inside the AnalysisScaffolding class.
- moved the more complex/badly formatted xmls (that were a mess to generate) to a specific folder called `SonarLintXml` to better define the scope. 

So if the CreateSonarLintXml method found on the AnalysisScaffolding is not able to generate the file we need, at least to me It's ok to add that inside the `SonarLintXml` folder.

Reply:
@pavel-mikula-sonarsource  Based on the answer above this file is ok to be there.